### PR TITLE
Mention ARM-based HW acceleration

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -236,7 +236,7 @@ \subsubsection{End-to-End-Encryption}
     \item Send the ciphertext along with signed MAC.
 \end{enumerate}
 
-The E2EE protocol presented in this paper combines a series of different cryptographic procedures. Each of them can be chosen on the basis of the working hardware and scenario by developers. On some certain devices, hardware acceleration would be supported. With hardware acceleration, the elapsed time could be dropped a step more. For example applying AES-NI can accelerate AES operations around 40\% on Intel devices.\cite{AES-NI-Acceleration}
+The E2EE protocol presented in this paper combines a series of different cryptographic procedures. Each of them can be chosen on the basis of the working hardware and scenario by developers. On some certain devices, hardware acceleration would be supported. With hardware acceleration, the elapsed time could be dropped a step more. For example applying AES-NI can accelerate AES operations around 40\% on Intel devices.\cite{AES-NI-Acceleration} On ARM-based devices have similar hardware acceleration implementation.\cite{stm}
```
Comment (jserv) [inline paper.tex]: s/stm/stm32/
最後一句話改為 "Some popular microcontrollers have integrated hardware accelerators for AES as well."

File: references.bib [hunk 2]
```diff
@@ -519,6 +519,12 @@ @electronic{AES-NI-Acceleration
   urldate = {2012-06-01}
 }
 
+@electronic{stm,
+  Author = {STMicroelectronics},
+  title = {UM1924 User manual STM32 crypto library},
+  url= "https://www.st.com/resource/en/user_manual/dm00215061-stm32-crypto-library-stmicroelectronics.pdf"
```
Comment (jserv) [inline references.bib]: Add last retrieval date.

**Comment by howjmay** [comment]
@jkrvivian could you help me to fix the latex compiling error?

**Comment by jkrvivian** [comment]
@HowJMay  Fixed!

File: references.bib [hunk 3]
```diff
@@ -519,6 +519,13 @@ @electronic{AES-NI-Acceleration
   urldate = {2012-06-01}
 }
 
+@electronic{stm,
+  Author = {STMicroelectronics},
+  title = {UM1924 User manual STM32 crypto library},
+  url= "https://www.st.com/resource/en/user{\_}manual/dm00215061-stm32-crypto-library-stmicroelectronics.pdf",
+  year={2020},
```
Comment (jserv) [inline references.bib]: 需要明確指出上次存取的日期 (年月日)

**Pull request merged**